wrangler-core/src/main/java/co/cask/wrangler/steps/ParseLog.java

```diff
@@ -28,6 +29,7 @@
 /**
  * A Step for parsing Apache HTTPD and NGINX log files.
```

Comment:
Since you can specify the format, this could be applied to any log files correct?